src/Api/Models/Request/Organizations/OrganizationSmSubscriptionUpdateRequestModel.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System.ComponentModel.DataAnnotations;
+using Bit.Core.Models.Business;
+
+namespace Bit.Api.Models.Request.Organizations;
+
+public class OrganizationSmSubscriptionUpdateRequestModel
+{
+    [Required]
+    public int SeatAdjustment { get; set; }
+    public int? MaxAutoscaleSeats { get; set; }
+
+    public int? ServiceAccountsAdjustment { get; set; }
```

Comment:
suggestion: Make this singular to match the `SeatAdjustment` above and in the other PM subscription update request model.
```suggestion
    public int? ServiceAccountAdjustment { get; set; }
```

Reply:
done